operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/spec/HostnameSpec.java

```diff
@@ -40,6 +40,7 @@ public class HostnameSpec implements Serializable {
     @JsonPropertyDescription("By default backchannel URLs are dynamically resolved from request headers to allow internal and external applications.")
     private Boolean strictBackchannel;
 
+
```

Comment:
I'd suggest to not make any changes to this file to reduce noise in this PR.

Reply:
Ok. Done!